pshtt/cli.py

```diff
@@ -107,9 +105,7 @@ def main():
     options = {
         'user_agent': args['--user-agent'],
         'timeout': args['--timeout'],
-        'preload_cache': args['--preload-cache'],
-        'suffix_cache': args['--suffix-cache'],
-        'cache': args['--cache'],
```

Comment:
The `--cache` flag has been unused for a long time -- it's a remnant of a time when this library used `requests-cache`, but that was removed a long time ago.

Reply:
For info on that, see https://github.com/reclosedev/requests-cache/issues/70.